recipes-demos/ti-apps-launcher/ti-apps-launcher.bb

```diff
@@ -10,8 +10,8 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 LICENSE = "TI-TFL"
 LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=5c3a7f5f6886ba6f33ec3d214dc7ab4c"
 
-DEPENDS = "qtbase qtquick3d qtdeclarative qtgraphicaleffects qtmultimedia qtxmlpatterns"
-RDEPENDS:${PN} = "qtquick3d qtmultimedia bash seva-launcher pulseaudio-service qtdeclarative-qmlplugins qtgraphicaleffects-qmlplugins"
+DEPENDS = "qtbase qtquick3d qtdeclarative qtgraphicaleffects qtmultimedia qtxmlpatterns qmltermwidget"
```

Comment:
@paresh-bhagat12 Are you planning to add qmltermwidget in j721s2|j784s4|j722s as well?
Since adding it under DEPENDS would eventually build your qmltermwidget recipe for those platforms as well